tutorindigo/templates/indigo/lms/templates/header/navbar-logo-header.html

```diff
@@ -20,14 +20,14 @@
 
 <h1 class="header-logo">
   % if enterprise_customer_link:
-    <a href="${settings.ENTERPRISE_LEARNER_PORTAL_BASE_URL}/${enterprise_customer_link.get('slug')}">
+    <a href="${settings.ENTERPRISE_LEARNER_PORTAL_BASE_URL}/${enterprise_customer_link.get('slug')}" title="${_('{name} Dashboard').format(name=enterprise_customer_link.get('name'))}">
       <img class="logo" src="${enterprise_customer_link.get('logo')}" alt="${_('{name} Dashboard').format(name=enterprise_customer_link.get('name'))}"/>
```

Comment:
Is this the right behavior to use the same value for <a> title and <img> alt?

Reply:
Dashboard replaced with Learner Dashboard